src/Entity/IncentiveImport.php

```diff
@@ -0,0 +1,169 @@
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $site;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $createdTs;
+
+    /**
+     * @ORM\Column(type="smallint", options={"default":0})
+     */
+    private $importStatus = 0;
+
+    /**
+     * @ORM\Column(type="smallint", options={"default":0})
+     */
+    private $confirm = 0;
+
+    /**
+     * @ORM\OneToMany(targetEntity="IncentiveImportRow", mappedBy="import", cascade={"persist", "remove"})
+     */
+    private $IncentiveImportRows;
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getFileName(): ?string
+    {
+        return $this->fileName;
+    }
+
+    public function setFileName(string $fileName): self
+    {
+        $this->fileName = $fileName;
+
+        return $this;
+    }
+
+    public function getUser(): ?User
+    {
+        return $this->user;
+    }
+
+    public function setUser(?User $user): self
+    {
+        $this->user = $user;
+
+        return $this;
+    }
+
+    public function getSite(): ?string
+    {
+        return $this->site;
+    }
+
+    public function setSite(string $site): self
+    {
+        $this->site = $site;
+
+        return $this;
+    }
+
+    public function getCreatedTs(): ?\DateTimeInterface
+    {
+        return $this->createdTs;
+    }
+
+    public function setCreatedTs(\DateTimeInterface $createdTs): self
+    {
+        $this->createdTs = $createdTs;
+
+        return $this;
+    }
+
+    public function getImportStatus(): ?int
+    {
+        return $this->importStatus;
+    }
+
+    public function setImportStatus(int $importStatus): self
+    {
+        $this->importStatus = $importStatus;
+
+        return $this;
+    }
+
+    public function getConfirm(): ?int
+    {
+        return $this->confirm;
+    }
+
+    public function setConfirm(int $confirm): self
+    {
+        $this->confirm = $confirm;
+
+        return $this;
+    }
+
+    /**
+     * @return Collection|IncentiveImportRow[]
+     */
+    public function getIncentiveImportRows(): Collection
+    {
+        return $this->IncentiveImportRows;
+    }
+
+    public function addIncentiveImportRow(IncentiveImportRow $IncentiveImportRow): self
+    {
+        if (!$this->IncentiveImportRows->contains($IncentiveImportRow)) {
+            $this->IncentiveImportRows[] = $IncentiveImportRow;
+            $IncentiveImportRow->setImport($this);
+        }
+
+        return $this;
+    }
+
+    public function removeIncentiveImportRow(IncentiveImportRow $IncentiveImportRow): self
+    {
+        if ($this->IncentiveImportRows->contains($IncentiveImportRow)) {
+            $this->IncentiveImportRows->removeElement($IncentiveImportRow);
+            // set the owning side to null (unless already changed)
+            if ($IncentiveImportRow->getImport()->getId() === $this->getId()) {
+                $IncentiveImportRow->setImport(null);
+            }
+        }
```

Comment:
Why is the `$IncentiveImportRow` variable capitalized?

Reply:
I think this is because it's mapped to IncentiveImportRow entity.

Reply:
Aren't all of our variable names always lowercase? I don't think we ever treat instances of entity objects any differently.

Reply:
This code is generated by maker bundle, I see that in other places too https://github.com/all-of-us/healthpro/blob/develop/src/Entity/PatientStatusImport.php#L163. Maybe we should change to lowercase?

Reply:
I thought the PSR standards addressed the question of variable capitalization but it turns out they purposely avoid it.  Nevertheless, let's make all method and variable names start with a lowercase letter, while class names will start with a capital letter.

Reply:
Turns out this capitalization rule is in the [Symfony Coding Standards](https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions).  That explains why I couldn't find them in the PSR.